tests/phpunit/tests/theme/wpGetGlobalStylesheet.php

```diff
@@ -54,6 +54,18 @@ public function tear_down() {
 		parent::tear_down();
 	}
 
+	/**
+	 * Cleans up global scope.
+	 *
+	 * @global WP_Scripts $wp_scripts
```

Comment:
Is it used `$wp_scripts` here?

Reply:
Ah, copy-pasta! Good catch 😅  Only `$wp_styles` it seems. I'll remove `$wp_scripts`.